### ISSUE TITLE
Can the SNOMED code '59000001' be moved to the Pcocedure domain?

### ISSUE THREAD

**Issue by mgurley** [OHDSI/OMOP-Standardized-Vocabularies#28]

**Comment by cgreich** [comment]
Well, the definition of a procedure currently is that it has to be done on the patient. Something that happens in a lab or path lab does not constitute a procedure (even though there are procedural aspects to the act). In other word "it has to hurt a little". Why do you care?

**Comment by mgurley** [comment]
@cgreich Would you say a 'Surgical Pathology Procedure' is a procedure?  SNOMED and OMOP say it is a procedure.  See here:

https://browser.ihtsdotools.org/?perspective=full&conceptId1=39228008&edition=us-edition&release=v20180901&server=https://prod-browser-exten.ihtsdotools.org/api/snomed&langRefset=900000000000509007

http://athena.ohdsi.org/search-terms/terms?query=39228008&page=1

'Surgical pathology consultation and report on referred slides prepared elsewhere'  is a child of 'Surgical Pathology Procedure'  in both SNOMED and OMOP.  So your argument would imply that all pathology procedure are not procedures. 
  We care because we are setting up a system that is analyzing pathology procedures and their connection to surgical procedures.  With relationships being made between the two in FACT_RELATIONSHIP (ugly but currently no other way).   We are setting up queries using pathology procedure codes in OMOP and don't want to have to make special case logic for outside pathology procedures.

**Comment by mgurley** [comment]
@cgreich 
Also currently CPT4 code 88321 'Consultation and report on referred slides prepared elsewhere' is in the Procedure domain in OMOP.
http://athena.ohdsi.org/search-terms/terms?query=Surgical+pathology+consultation+and+report+on+referred+slides+prepared+elsewhere&page=1

**Comment by dimshitc** [comment]
Oh, I see there's an inconsistency. 
@cgreich should we put all descendants of 4213297 "Surgical pathology procedure" and related HCPCS, CPT4 concepts put into Observation?

@mgurley , for you, if you know that all such a concepts belong to the Observation domain, will it be OK?

**Comment by mgurley** [comment]
@dimshitc @cgreich Yes, I am mostly concerned that all codes that identify pathology procedures should belong to the same domain.  The detour for the pathology procedure on outside slides seems arbitrary.  My specific technical dilemma is to avoid special logic for outside versus inside pathology procedures.

However, I do believe most folks would assume that the pathology examination of specimen is a 'Procedure'.  So putting all pathology procedures in the Observation would buck what I would take to be most folks intuition/common sense.   My guess is that if you throw this question out to the Forums most people would be that if your definition of a procedure make pathology procedures observations then something is wrong with your definition of a procedure.  Then again, that could just be my own crank opinion.

**Comment by cgreich** [comment]
Gents:

We can change the convention and make path jobs procedures. Or observations. I think it is in the grey zone between the two. However, as @dimshitc says, it should be consistent.

**Comment by mgurley** [comment]
@cgreich @dimshitc Yes, consistency is most important.  

But the definition of the PROCEDURE_OCCURRENCE is the following:

"The PROCEDURE_OCCURRENCE table contains records of activities or processes ordered by, or carried out by, a healthcare provider on the patient to have a diagnostic or therapeutic purpose. Procedures are present in various data sources in different forms with varying levels of standardization/"

https://github.com/OHDSI/CommonDataModel/wiki/PROCEDURE_OCCURRENCE

To my mind, a pathology examination is a paradigmatic diagnostic activity/process.  Yes, you could quibble that a specimen from the patient is not the patient.   But to my mind a pathology examination of a patient specimen is a procedure that yields measurements and observations.  The examination itself is not a measurement or observation.  

So can we go with my original request? Move SNOMED code '59000001' to the Procedure domain.

**Comment by dimshitc** [comment]
yes, it totally makes sense that Pathology is a diagnostic procedure.
So, reports belong to Procedure domain, and what's written in those reports go to NOTE table connected through FACT_RELATIONSHIP. Right?

on the other hand, there is exact example with Pathology report here
http://www.ohdsi.org/web/wiki/doku.php?id=documentation:vocabulary:domains

Pathology report | HCPCS G9428 Pathology report includes the pt category and a statement on thickness and ulceration and for pt1, mitotic raten | Observation
-- | -- | --

**Comment by dimshitc** [comment]
Well, it's total mess now:
```
select domain_id, count(1) from concept where lower (concept_name) like '%pathology report%' and standard_concept ='S' 
group by domain_id 
;
```

Measurement | 18
-- | --
Observation | 13
Procedure | 5
Meas Value | 1
Type Concept | 1

**Comment by cgreich** [comment]
Let's put this to bed. Can't be a Measurement because it cannot have a single result. I don't have a strong opinion if it is an Observation or Procedure. Don't like Observation ("garbage can", nobody trusts it), or Procedure (it really isn't a procedure on a patient). After flipping a coin let's make it an Observation.

**Comment by mgurley** [comment]
A 'Procedure' would make it fall in line with all other pathology procedures.  So can we make it a 'Procedure'?  This will simplify our use cases by allowing us only look within only one domain for pathology procedure.

**Comment by dimshitc** [comment]
@mgurley, seems that @cgreich wants to move every Pathology procedure to Observation domain, so you will need to look on one domain only

**Comment by mgurley** [comment]
@dimshitc @cgreich Is that what you are suggesting?  That we move 'Anatomic pathology procedure' and all concepts subsumed under it into 'Observation'?  
See here http://athena.ohdsi.org/search-terms/terms/4028909
That suggestion was actually my reductio ad absurdum  to just get this one concept (SNOMED code '59000001') moved from 'Obseration' into 'Procedure'.

If so, I think  for sure we should throw this out to the community.  My guess is that most people would consider anatomical pathology procedures as belonging in the 'Procedure' domain.  Yes, an anatomic pathology procedure is an activity performed on a specimen extracted from an patient. So strictly speaking not **ON** a patient. But that is too narrow a definition of 'Procedure'.  Makes me think Wittgenstein's concept of Family Resembelance: https://en.wikipedia.org/wiki/Family_resemblance

**Comment by cgreich** [comment]
Well, I have no ideological base for this choice. Pragmatism is key. And what the community "considers" a procedure might be a good criterion. Except I don't think it is, because there is a slippery slope to lab procedures, or all sorts of services. In a way, a transportation in an ambulance is more of a procedure carried out on the patient than what they do in the path lab. 

Currently, the best distinguishing criterion we have is activity **on** the patient vs not on the patient. That is pretty straightforward, and if the community considered this criterion they know where to look for things.

**Comment by mgurley** [comment]
Methinks the definition needs to be aligned with conventional wisdom:
"The PROCEDURE_OCCURRENCE table contains records of activities or processes ordered by, or carried out by, a healthcare provider on the patient, **OR A SPECIMEN EXTRACTED FROM THE PATIENT**, to have a diagnostic or therapeutic purpose. Procedures are present in various data sources in different forms with varying levels of standardization.

**Comment by dimshitc** [comment]
Oh, I like this.
@cgreich ?

**Comment by cgreich** [comment]
As I said, I am no ideolog. If "Me" (=@mgurley) thinks that way - I am all for it. Procedures they are. :)

**Comment by dimshitc** [comment]
https://forums.ohdsi.org/t/how-to-store-pathological-examination-result-in-cdm/8032/4
yes, they are procedures. 
So, I put this into vocabulary team backlog.
